.github/workflows/ci-pipeline.yml

```diff
@@ -157,13 +157,21 @@ jobs:
       # Approved by the 'github-actions' user; a PR can't be approved by its author
       - name: PR approval
         if: ${{ needs.vib-verify.result == 'success' }}
-        uses: hmarr/auto-approve-action@v3.0.0
+        uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410
         with:
-          pull-request-number: ${{ github.event.number }}
+          result-encoding: string
+          retries: 3
+          script: |
+            github.rest.pulls.createReview({
```

Comment:
Probably you have tested it already (and in case you don't because of permissions, we should be extra-careful when landing this), but sometimes in the past we have faced limitation in order to approve a PR because of the actor (i.e. the creator of the PR can't approve its own PR, so we had to use two different tokens). 

I don't see any mention to tokens here (neither before the change, but maybe it was something intrinsic of the action we used), but just as a reminder to us :)

Reply:
Thanks a lot for the heads up, that's a good one. Automated PRs are created by `bitnami-bot` so the PR should be  approved by other user (this is configured in our repo to avoid self approvals) In this piece of code there is no reference to the token because it uses by default GitHub Actions user and we are allowing write permissions to pull-requests in that job: https://github.com/bitnami/charts/blob/c4b617b6b0ce71e438994bd35864a107f3026b44/.github/workflows/ci-pipeline.yml#L147-L148